ion-schema/Cargo.toml

```diff
@@ -28,6 +28,7 @@ num-bigint = "0.3"
 num-traits = "0.2"
 chrono = "0.4"
 regex = "1.5.6"
+half = "2.2.1"
```

Comment:
Nice! I'm glad we didn't have to have some hokey thing like in `ion-schema-kotlin`.